pkg/scaffold/v1/metricsauth/kustomize_auth_proxy_patch.go

```diff
@@ -40,7 +40,8 @@ func (c *KustomizeAuthProxyPatch) GetInput() (input.Input, error) {
 	return c.Input, nil
 }
 
-const kustomizeAuthProxyPatchTemplate = `# This patch inject a sidecar container which is a HTTP proxy for the controller manager,
+const kustomizeAuthProxyPatchTemplate = `
+# This patch inject a sidecar container which is a HTTP proxy for the controller manager,
```

Comment:
This changes the generated file as it introduces a new line at the start of the file. IMHO you should mark it `// nolint:lll` and leave it as is.